tests/unit/processing_container/test_process.py

```diff
@@ -45,24 +44,39 @@ def download(url, local_filename):
     @classmethod
     def tearDownClass(cls):
         super().tearDownClass()
-        shutil.rmtree(cls.tmp_dir)
+        cls.clean_generated_files()
 
-    def test_make_cxg(self):
-        make_cxg(str(self.real_h5ad_filename))
-
-    def test_make_seurat(self):
-        make_seurat(str(self.real_h5ad_filename))
+    @classmethod
+    def clean_generated_files(cls):
+        if os.path.exists("local.cxg"):
+            shutil.rmtree("local.cxg")
+        for f in ["local.h5ad", "local.loom", "local.rds"]:
+            if os.path.exists(f):
+                os.remove(f)
 
-    def test_make_loom(self):
-        make_loom(str(self.real_h5ad_filename))
+    @patch("backend.corpora.dataset_processing.process.download_from_dropbox_url")
+    def test_main(self, mock_download_from_dropbox):
+        """
+        Tests full pipeline for processing an uploaded H5AD file, including database updates
+        generation and upload of all artifacts to S3 (localstack), but excluding the Dropbox download
+        functionality.  Dropbox I/O is is mocked to prevent dependency on remote services (non-Dockerized).
+        """
+        mock_download_from_dropbox.return_value = self.h5ad_raw
 
-    # TODO: Remove mocking of validate_h5ad_file after validation process is updated to be 2.0 compliant.
-    @patch("backend.corpora.dataset_processing.process.validate_h5ad_file_and_add_labels")
-    def test_main(self, mock_validate_h5ad):
-        mock_validate_h5ad.return_value = "raw.h5ad"
-        url = self.presigned_url
         dataset = self.generate_dataset(
             self.session, collection_id="test_collection_id", collection_visibility=CollectionVisibility.PUBLIC.name
         )
-        self.bucket
-        process.process(dataset.id, url, self.corpora_config.bucket_name, self.corpora_config.bucket_name)
+
+        process.process(
+            dataset.id,
+            "https://www.dropbox.com/IGNORED",
+            self.corpora_config.bucket_name,
+            self.corpora_config.bucket_name,
+        )
+
+        # TODO: add assertions:
```

Comment:
Nit: Is there a future story to track this?

Reply:
added!